test/common/testlib.py

```diff
@@ -731,21 +732,88 @@ def post_github(self, number, err):
         if not github.available:
             return False
 
+        timestamp = datetime.datetime.now().isoformat()
+
         # Lookup the link being logged to
         context = testinfra.DEFAULT_IMAGE
-        link = ""
+        link = timestamp
         revision = os.environ.get("TEST_REVISION", None)
         if revision:
+            link = "{0}: revision {1}".format(timestamp, revision)
             statuses = github.get("commits/{0}/statuses".format(revision))
             if statuses:
                 for status in statuses:
                     if status["context"] == context:
-                        link = status["target_url"]
+                        link = "[{0}]({1})".format(timestamp, status["target_url"])
+                        break
+
+        comments = github.get("issues/{0}/comments".format(number))
```

Comment:
> Doesn't github page comments? Does this need a loop and ?page=x&per_page=100?

either that or something else is happening, too: https://github.com/cockpit-project/cockpit/issues/4001#issuecomment-199257031 has duplicate posts